angular-workspace/components/annotation/annotation.content.ts

```diff
@@ -0,0 +1,33 @@
+import { Templates } from "templates";
+
+export function annotationContent({ selectableTemplate }: Templates) {
+  return {
+    title: { template: `<h2 slot="title">Annotaties</h2>` },
+    addons: {
+      template: selectableTemplate(
+        {
+          type: "checkbox",
+          id: "123",
+          label: "Toon uitgebreide weergave",
+          slot: "addons",
+          value: "",
+        },
+        {
```

Comment:
Dit is jammer maar gezien de staat van onze angular templates is het maar even zo.